index.es.md

```diff
@@ -0,0 +1,113 @@
+-   El W3C está desarrollando requisitos actualizados y más específicas guías de accesibilidad móvil.
+-   **El resultado del [Grupo de Trabajo de Accesibilidad Móvil](https://www.w3.org/WAI/GL/mobile-a11y-tf/) de la WAI incluye:**
+    -   [Accesibilidad Móvil: Cómo las WCAG 2.0 y otras pautas del W3C/WAI se aplican al entorno Móvil](http://www.w3.org/TR/mobile-accessibility-mapping/)
+    -   Criterios de Éxito y Técnicas WCAG 2.1
+    -   Integrar la accesibilidad móvil en los futuros estándares W3C
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+
+{::options toc_levels="2" /}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Contenido de la Página" %}
+{:/}
+
+-   TOC is created automatically.
+{:toc}
+
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+## Más que "Móvil" {#intro}
+
+La "accesibilidad móvil" se refiere a hacer los sitios web y aplicaciones más accesibles para personas con discapacidad cuando utilizan teléfonos móviles y otros dispositivos. El trabajo de la WAI en este área aborda problemas de accesibilidad para personas utilizando un amplio rango de dispositivos para interactuar con la web:
+
+-   teléfonos y tabletas
+-   televisores digitales
+-   accesorios como los teléfonos inteligentes
+-   dispositivos en salpicaderos de los automóviles y en los respaldos de los asientos de los aviones
+-   dispositivos en electrodomésticos
+-   "Internet de las cosas"
+
+Tsmbién considera una amplia gama de cuestiones:
+
+-   pantallas táctiles
+-   pantallas de reducidas dimensiones
+-   diferentes métodos de entrada, incluyendo la voz y el tacto 3D habilitado por sensores de presión
+-   dispositivos afectados por el uso dependiente de diferentes entornos, como con el brillo del sol
+-   y más
+
+## W3C WAI Considera la Accesibilidad Móvil {#covered}
+
+**Los estándares de accesibilidad WAI consideran la accesibilidad móvil**:
+
+-   Las Pautas de Accesibilidad del **Contenido Web** ([WCAG](/standards-guidelines/wcag/)) consideran páginas y aplicaciones web, incluyendo el contenido utilizado en dispositivos móviles.
+    -   Para aprender cómo las WCAG 2.0 pueden aplicarse al contenido web móvil, aplicaciones web móviles, aplicaciones nativas e híbridas utilizando componentes dentro de las aplicaciones nativas, acceda a [Accesibilidad Móvil: Cómo las WCAG 2.0 y Otras Pautas W3C/WAI Aplican al entorno Móvil](http://www.w3.org/TR/mobile-accessibility-mapping/).
+    -   [WCAG2ICT: Aplicar WCAG 2.0 a Información No-Web y Comunicaciones Tecnológicas](http://www.w3.org/TR/wcag2ict/) es otro recurso más general que también incluye aplicaciones móviles.
+    -   Las WCAG 2.1, publicadas en junio de 2018, incluyen nuevos requisitos ("criterios de éxito") que tienen en consideración la accesibilidad móvil. Se introducen en [[Novedades de las WCAG 2.1]](/standards-guidelines/wcag/new-in-21/).
```

Comment:
Proposal:
`    -   Las WCAG 2.1, publicadas en junio de 2018, incluyen nuevos requisitos ("criterios de éxito") que tienen en consideración la accesibilidad móvil. Se presentan en [[Novedades de las WCAG 2.1]](/standards-guidelines/wcag/new-in-21/).`

(s/introducen/presentan)